misc/scripts/header_guards.py

```diff
@@ -32,138 +31,53 @@
                 break
         else:
             if not sline.startswith("*") and not sline.startswith("/*"):  # Not in the Godot header anymore.
-                HEADER_CHECK_OFFSET = idx + 1  # The include should be two lines below the Godot header.
+                HEADER_CHECK_OFFSET = idx + 1  # The guard should be two lines below the Godot header.
```

Comment:
```suggestion
                header_check_offset = idx + 1  # The guard should be two lines below the Godot header.
```
The Pythonic way is to use uppercase for constants, however `HEADER_CHECK_OFFSET` is modified (unlike `HEADER_BEGIN_OFFSET` & `HEADER_END_OFFSET` that are defined once and never modified)